src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -22,12 +24,16 @@
  * @author Jesus Menacho
```

Comment:
write again your documentation, must be written in third person

Reply:
That is write in third person.